src/submission-handlers/patientIdentifierHandler.ts

```diff
@@ -24,8 +23,7 @@ export const PatientIdentifierHandler: SubmissionHandler = {
     allFormFields: Array<FormField>,
     context: EncounterContext,
   ) => {
-    const latestIdentifier = findLast(
-      context.patient.identifier,
+    const latestIdentifier = context.patient.identifier.find(
```

Comment:
switched to inbuilt` find()` since it was not returning a plain string but rather a function or a wrapper around the actual value during testing